packages/core/content-releases/admin/src/modules/releaseSlice.ts

```diff
@@ -5,17 +5,36 @@ import { axiosBaseQuery } from '../utils/data';
 
 import type { CreateRelease, GetAllReleases } from '../../../shared/contracts/releases';
 
+interface GetAllReleasesQueryParams {
+  page?: number;
+  pageSize?: number;
+  filters?: {
+    $and?: Array<{
+      releasedAt?: {
+        $notNull?: boolean;
+      };
+    }>;
+  };
+}
```

Comment:
I'm wondering if we should make the update to the GetReleases.Request type contract. Technically we can send any query the entity service accepts. I guess maybe it will be sort of generic there and we could narrow further on the frontend 🤔 .

I can make the update over here: https://github.com/strapi/strapi/pull/18867. Otherwise I think for now you could at least extend from the existing type contract which I think includes page? pageSize? already

Reply:
Do you mean to import Pagination and pick pageSize and page from it?